nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -830,7 +830,7 @@ Returns a faction datum by its name (case insensitive!)
 	is_capital = TRUE
 	x = 70
 	y = 50
-	fleet_type = /datum/fleet/nanotrasen/earth
+	fleet_type = /datum/fleet/solgov/earth
 	alignment = "nanotrasen"
```

Comment:
do change the alignment to be "solgov" as well

Reply:
A bit messy, but done